src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/Methodologies/MethodologyRepositoryTests.cs

```diff
@@ -0,0 +1,310 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Admin.Services.Methodologies;
+using GovUk.Education.ExploreEducationStatistics.Content.Model;
+using Xunit;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Tests.Services.Methodologies
+{
+    public class MethodologyRepositoryTests
+    {
+        [Fact]
+        public async Task UserHasReleaseRoleAssociatedWithMethodology_ReleaseRoleLead()
+        {
+            var userId = Guid.NewGuid();
+            var publication = new Publication
+            {
+                Id = Guid.NewGuid()
+            };
+            var release = new Release
+            {
+                Id = Guid.NewGuid(),
+                PublicationId = publication.Id
+            };
+            var userReleaseRole = new UserReleaseRole
+            {
+                UserId = userId,
+                Release = release,
+                Role = ReleaseRole.Lead,
+            };
+            var methodology = new Methodology
+            {
+                Id = Guid.NewGuid(),
+                Title = "Test methodology",
+                Publications = new List<Publication> {publication}
+            };
+
+            var contentDbContextId = Guid.NewGuid().ToString();
+            await using (var contentDbContext = DbUtils.InMemoryApplicationDbContext(contentDbContextId))
+            {
+                await contentDbContext.AddAsync(publication);
+                await contentDbContext.AddAsync(release);
+                await contentDbContext.AddAsync(userReleaseRole);
+                await contentDbContext.AddAsync(methodology);
+                await contentDbContext.SaveChangesAsync();
+            }
+
+            await using (var contentDbContext = DbUtils.InMemoryApplicationDbContext(contentDbContextId))
+            {
+                var methodologyRepository = new MethodologyRepository(contentDbContext);
```

Comment:
Would instantiate our repositorites using a `BuildMethodologyRepository` method (like how we do in services). If we ever need to add more dependencies, this can be really easily done without modifying all call sites.